content/using/install.md

```diff
@@ -151,3 +151,19 @@ Landscape is the Urbit web interface, and it's the best way to interact with you
 2.  If the port given is 80, simply type `localhost` into your browser's address bar. If the given port is a different number, such as `8080`, you would type `localhost:8080`. Alternatively, you access landscape by navigating to `ship.arvo.network` into your URL bar, where `sampel-palnet` is the name of your ship, without the `~`. You will need to [set up DNS proxying](@/using/operations/using-your-ship.md#dns-proxying) to use this option.
 3. Type `+code` into your ship's Dojo. Copy-paste the returned code into the field asking for it in the browser page.
 4. You're in! Now you can explore apps such as Chat for messages, Publish for blogging, and Weather.
+
+### Joining the community
+
+Once you are in Landscape we recommend joining the Urbit Community group. You
+will need to obtain an invite, which can be had by first joining
+the `~/~dopzod/urbit-help` chat channel and asking for an invitation from there.
```

Comment:
Wonder if we should leave a disclaimer here that it's more likely to have a request fulfilled during West Coast US hours

Reply:
Correction: 

> Wonder if we should leave a disclaimer here [Line 159] that one is most likely to have an invite request fulfilled during West Coast US hours.

Reply:
idk, I'm generally on during UTC hours, but I'm also just one person instead of the army we have on US hours. (Generally opposed to doing and/or signalling timezone favoritism, but no way around the fact of the situation...)

Reply:
Yeah, I wasn't forgetting you – just wanted to be sure we weren't putting a huge load of responsibility on a single individual or placing that expectation or making that promise, no need to warn about this if you don't feel it's necessary